### PR TITLE
add drop and char-mods callback

### PR DESCRIPTION
**Pull request by gregcman**

### CODE REVIEW

**Comment by borodust** [comment]
[`glfwSetCharModsCallback`](https://www.glfw.org/docs/latest/group__input.html#ga3f55ef5dc03a374e567f068b13c94afc) is deprecated.

File: glfw.lisp [hunk 1]
```diff
@@ -61,6 +61,17 @@
   `(claw:defcallback ,name :void ((,joystick-id :int) (,event-id :int))
      ,@body))
 
+(defmacro define-drop-callback (name (window count paths) &body body)
+  `(claw:defcallback ,name :void ((,window (:pointer %glfw:window))
+                                  (,count :int)
+				  (,paths (:pointer (:pointer :char))))
```
Comment (borodust) [inline glfw.lisp]: Please, use spaces instead of tabs

**Comment by gregcman** [comment]
ok, how about drop callback?

**Comment by borodust** [comment]
Drop callback looks good.

File: glfw.lisp (hunk 1 above)
Comment (gregcman) [reply glfw.lisp]: ok, see https://github.com/borodust/bodge-glfw/pull/1/commits/d6d87bc5aaaef90ded9ac273f58426d9175b982e

**Comment by borodust** [comment]
Please, squash commits into one, fix commit message and we are done, I'll merge it :)

**Comment by borodust** [comment]
Actually, nvm, I'll rebase it myself. Thanks for the PR.

**Comment by borodust** [comment]
Testing dist updated.

**Pull request merged**